codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -434,6 +434,7 @@ public void onSettingsRead(ChannelHandlerContext ctx, Http2Settings settings) th
 
             // Acknowledge receipt of the settings.
             encoder.writeSettingsAck(ctx, ctx.newPromise());
+            ctx.flush();
```

Comment:
Is this and the `onPingRead` just to be more responsive for setting up connections and responding to pings?